guides/v2.2/coding-standards/code-standard-javascript.md

```diff
@@ -14,15 +14,14 @@ Use [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt) to interpret the "must," "mu
 Magento uses the [jQuery library][jquery] including standard and custom [jQuery widgets][jquery-widgets].
 For the jQuery widget coding standard, see [jQuery widget coding standard][jquery-widget-coding-standard].
 
-## Eslint and JSCS tools
+## Eslint tool
 
-Use [ESLint][eslint] and [JSCS][jscs] to ensure the quality of your JavaScript code.
+Use [ESLint][eslint] to ensure the quality of your JavaScript code.
 
 ESLint is a community-driven tool that detects errors and potential problems in JavaScript code.
 It can use custom rules to enforce specific coding standards.
 
 * [Magento ESLint Rules][eslint-rules]
-* [Magento JSCS Rules][jscs-rules]
```

Comment:
Thanks @lenaorobei for confirming. @brunoroeder since Magento still has the JSCS rules, let's not remove the link to the `Magento JSCS Rules`.